src/main/java/codeu/bot/BotActions.java

```diff
@@ -153,19 +204,36 @@ public void doAction(Object ... argsObjects) {
                 String content= "All conversations have been retrieved";
                 addAnswerMessageToStorage(titleConversations + content);
             }
-        },/*
+        },
         GET_STATS{
             @Override
             public void doAction(Object ... argsObjects) {
-                
+                List<Message> userMessages = messageStore.getMessagesByAuthor(user.getId());
+                int conversationCount = getConversationCount(userMessages);
+                userMessages.sort(Message.messageComparator);
+                Message longestMessage = userMessages.get(userMessages.size()-1);
+                long maxCharactersLength = longestMessage.getContent().length()-1;
+                String content = "This are your stats\n";
+                content += "Amount of messages: " + userMessages.size() + "\n";
+                content += "Conversations you have participated: " + conversationCount + "\n";
+                content += "Longest message with " + maxCharactersLength + " characerts\n";
+                addAnswerMessageToStorage(content);
             }
+
+			private int getConversationCount(List<Message> userMessages) {
```

Comment:
this seems to be defined twice. Not sure if they are exactly the same of not.

Reply:
@ricglz0201 can you fix this, I'm not sure which one you want to keep.

Reply:
Yep checking it later